pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/clustercontrol/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: layout.pug
+navigationTitle: Cluster Control
+title: Cluster Control
+menuWeight: 30
+excerpt: Add cluster controllers to set up the cluster control plane.
+beta: false
+enterprise: false
+---
+
+## Adding cluster controllers
```

Comment:
We don't use gerunds in headings.

Reply:
Can we define a linter rule for this?

Reply:
yeah that's a good idea. Let me ask @vacas5

Reply:
here's the docs for Vale. Not sure how we seeded the initial configuration, but I bet it's possible: https://docs.errata.ai/vale/styles